apps/src/code-studio/pd/workshop_dashboard/attendance/session_attendance.jsx

```diff
@@ -27,7 +27,6 @@ const SessionAttendance = React.createClass({
     workshopId: PropTypes.number.isRequired,
```

Comment:
Me too, but since this is already a large change I'd like to keep it from getting even larger and do that separately / later. As I said in the description I ran through the common scenarios manually.